src/framework/core/js/Fluid.js

```diff
@@ -352,8 +354,11 @@ var fluid = fluid || fluid_3_0_0;
      * @param {Any} totest - The value to be tested
      * @return {Boolean} `true` if the supplied value is an array
      */
+    // Note: The primary place jQuery->Array conversion is used in the framework is in dynamic components with a jQuery source.
     fluid.isArrayable = function (totest) {
-        return totest && (totest.jquery || Object.prototype.toString.call(totest) === "[object Array]");
+        return (Boolean)(totest && (Object.prototype.toString.call(totest) === "[object Array]"
+            || totest.jquery && totest.constructor && totest.constructor.prototype
+               && totest.constructor.prototype.jquery));
```

Comment:
It's a bit difficult to read. I wonder if we could either just use [`instanceOf`](https://stackoverflow.com/a/1853246/10198026) or make a new function that determines if something is a jQuery and make use of that here.

Reply:
Sure. We can't use instanceOf because jQuery might not be loaded at all (if we are on the server)

Reply:
As well as, of course, the possibility that the jQuery object originates from a different iframe